### ISSUE TITLE
Transition skips to end when interrupted 

### ISSUE THREAD

**Issue by ChrisArasin** [lightning-js/blits#170]
I think for Transitions to be useful, they should be interruptible without skipping to the end (whether by default or through some property). 

In the case of a user scrolling a row of tiles, a quick right then left press you would expect the row to start scrolling right, stop where it is when the user presses left, then go back to where it came from. You would not what it to jump to the end of the initial right-directed transition.

Quickly pressing right/left in the example below shows what I mean. 

https://playground.lightningjs.io/#2Ass8WOF4aJM

https://github.com/user-attachments/assets/24d1f24a-260b-440e-9e61-e09917001566

Just noting the behavior seems a bit consistent as well, I've sometimes seen the transition not jump to the end when interrupted.

**Comment by ChrisArasin** [comment]
Example on a row.

https://github.com/user-attachments/assets/7694ef5d-a7c5-41c0-bd28-8f219c01b7fb

**Comment by michielvandergeest** [comment]
Hey @ChrisArasin, thanks for your clear examples, showcasing the issue. And I agree that looks weird indeed! 

So, if I remember correctly this fast forward behaviour got added to fix a specific transition bug we encountered. But at this moment I’m not sure if it was ultimately just a work around for a previous issue in the way the renderer was handling animations, where we would sometimes have dangling animation promises ..

I’ll do some testing with the fast forward disable and see if that breaks anything.

Now I’m not a UX designer, but I’m imagining that fast forward would actually be more the exception than the rule? Is there ever a situation when starting a new transition on a prop, would require to first do a fast forward to the end value of the running transition? That kind means building in a deliberate “yank”, I guess. Again, my UX imagination on this is limited haha .. so pinging @DouweCnossen to chime in on this one as well

If there is no valid use case, then I think we should just remove it. We could consider a config option, like you suggested. But I prefer not too, if it’s super rare to use anyway.
